internal/driver/dummy/dummy.go

```diff
@@ -62,7 +62,7 @@ func (d *dummyOBM) DialConsole() (io.ReadCloser, error) {
 }
 
 func (d *dummyOBM) PowerOff() error {
-	log.Println("Powering off:", d)
+	log.Println("Powering off: %v", d)
```

Comment:
What was the reasoning for this? If you want to use format specifiers, you need to use `Printf`, rather than `Println`. The latter automatically does the equivalent with each of its arguments though.

Reply:
The `tip` build seems to actually be complaining about this, noting the superfluous %v: https://travis-ci.org/CCI-MOC/obmd/jobs/367794057#L492

Reply:
That was leftover from my testing, I'll remove it.